src/js/data/names.js

```diff
@@ -3436,7 +3436,7 @@ let first = {
         ["A.C.", 5],
         ["A.J.", 63],
         ["A.W.", 64],
-        ["ATorri", 65],
```

Comment:
Get rid of the manually edited names, I'm not sure if any of them are actually problematic. "Mis-spelled" names exist in reality too.